daemon/prune.go

```diff
@@ -25,6 +25,27 @@ var (
 	// ErrPruneRunning is returned when a prune request is received while
 	// one is in progress
 	ErrPruneRunning = fmt.Errorf("a prune operation is already running")
```

Comment:
I am also wondering if we could disable the export of error `ErrPruneRunning `? 
Since it is only used here.
But maybe out of the scope of this PR. 🐱